UnitTesting/UnitTest1.cs

```diff
@@ -14,5 +17,144 @@ public void FirstIdMonitor()
             Results val = new Results();
             Assert.AreEqual(1, val.getFirstID());
         }
+
+        [Test]
+        public void TestCalculateDailyWastage()
+        {
+            var build = new ConfigurationBuilder();
+            IConfiguration _config = build.Build();
+            var expected = GetTestExpectedDaily();
+            var dataset = GetTestDataDaily();
+            Procedures proc = new Procedures(new DatabaseContext(),_config);
+
+            Assert.IsTrue(expected.Equals((proc.CalculateDailyWastage(dataset))[0]));
```

Comment:
Doesn't the `Assert` class have an `Equal` function or something equivalent?

Reply:
I have a custom class that is evaluated and the Assert Equal doesn't play well with the custom object. I created my own Equals method inside said custom class that checks the list of struct that exist within this class and returns false if a single record within the list is not equal to it.